### PR TITLE
Fix #180414 inkorr.com

### LINKED ISSUE #180414: inkorr.com
### Issue URL (Annoyance)
 [https://inkorr.com/ru/skolko-v-srednem-zarabatyvaet-batuska-v-ukraine-212301](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Finkorr.com%2Fru%2Fskolko-v-srednem-zarabatyvaet-batuska-v-ukraine-212301)
Username: `@spok1`

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2 </summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3 </summary>

  Screenshot 3
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Android v3.6.11 (54)
Browser: | Chrome
License type: | paid
AdGuard mode: | VPN
HTTPS filtering: | enabled
Stealth mode options: | Hide your search queries, <br>Hide your Referrer from third-parties, <br>Hide your User-Agent
DNS filtering: | server: `https://dns.google/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard Ukrainian
Userscripts: | disabled

### PR DESCRIPTION
**Pull request by vityevato**
Removes annoyance about `Subscrime on Telegram channel` on `inkorr.com`

### CODE REVIEW

File: AnnoyancesFilter/Widgets/sections/widgets.txt [hunk 1]
```diff
@@ -2261,7 +2261,9 @@ samlab.ws##iframe[src^="https://yoomoney.ru/"]
 !
 hromadske.ua#$#.DonateModal-overlay { display: none !important; }
 hromadske.ua#$#body { overflow: visible !important; }
-!
+inkorr.com##.sub-tg-wrapper
```
Comment (zloyden) [inline AnnoyancesFilter/Widgets/sections/widgets.txt:2264]: That will block the social links in the footer. Better do another rule and add it to `annoyances.txt` - under this comment `! Annoying elements, related to social networks, which we do not block in the main Social filter`.

Comment (piquark6046) [inline AnnoyancesFilter/Widgets/sections/widgets.txt:2264]: Should be located in `AnnoyancesFilter/Popups/sections/subscriptions_specific.txt`.

File: AnnoyancesFilter/Widgets/sections/widgets.txt [hunk 2]
```diff
@@ -2261,7 +2261,9 @@ samlab.ws##iframe[src^="https://yoomoney.ru/"]
...
+inkorr.com##.views-row:has(.fa-telegram)
```
Comment (zloyden) [inline AnnoyancesFilter/Widgets/sections/widgets.txt:2265]: Add it to the `subscriptions_specific.txt`.

Comment (piquark6046) [inline AnnoyancesFilter/Widgets/sections/widgets.txt:2265]: Should be located in `AnnoyancesFilter/Popups/sections/subscriptions_specific.txt`.
Also, please make more specific like `.view-content .views-row:has(.fa-telegram)`.

File: AnnoyancesFilter/Widgets/sections/widgets.txt [hunk 3]
```diff
@@ -2261,7 +2261,9 @@ samlab.ws##iframe[src^="https://yoomoney.ru/"]
...
+inkorr.com##.tg-box
```
Comment (piquark6046) [inline AnnoyancesFilter/Widgets/sections/widgets.txt:2266]: Should be located in `AnnoyancesFilter/Popups/sections/subscriptions_specific.txt`.

**Review by piquark6046 — DISMISSED** [review]

File: AnnoyancesFilter/Widgets/sections/widgets.txt (hunk 3 above)
Comment (zloyden) [inline AnnoyancesFilter/Widgets/sections/widgets.txt:2266]: Looks like an ad. Add it to the `CyrillicFilters - common-sections - specific.txt`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Other/sections/annoyances.txt [hunk 4]
```diff
@@ -3195,4 +3195,6 @@ adguard.info,adguard.com,adguard.app##.hello_from_adguard_ext
 ! Detect Advanced Protection of AdGuard for iOS
 !+ PLATFORM(ios) NOT_OPTIMIZED
 adguard.info,adguard.com,adguard.app#$#.hello_from_adguard_advanced_protection_ios { display: none !important; }
-!
+! Annoying elements, related to social networks, which we do not block in the main Social filter
+inkorr.com##.sub-tg-wrapper
```
Comment (zloyden) [inline AnnoyancesFilter/Other/sections/annoyances.txt:3199]: Better to change it to smth like this.
```adblock
inkorr.com##.content--main  div.sub-tg-wrapper
```
Add just the rule without the comment here)
<details><summary>Screenshot:</summary>

image

</details><br/>

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**